tool-plugins/intellij/getting-started/plugin-features/README.md

```diff
@@ -12,53 +30,23 @@ You can debug Ballerina main/service programs with only few clicks as well.
 
 ![alt text](images/DebugSupport.gif)
 
-## Import packages on the fly
+## Import modules on the fly
 
-This feature adds import declarations on the fly. All you need to do is selecting the package name from the lookup list and the package declaration will be added automatically.
+This feature adds import declarations on the fly. All you need to do is selecting the module name from the lookup list and the module declaration will be added automatically.
 
 ![alt text](images/AutoImports.gif)
 
-## Auto import unambiguous packages
+## Auto import unambiguous modules
```

Comment:
Comment on title case applies here as well. Let's go with title case everywhere so that it is consistent.